tasks/main.yml

```diff
@@ -2,6 +2,27 @@
 - name: Setup firewalld
   include_tasks: firewalld.yml
 
+- name: Collect Service Facts
+  service_facts:
+  when: firewall_disable_conflicting_services
+
+- name: Attempt to stop and disable conflicting services
+  service:
+    name: "{{ item }}"
+    state: stopped
+    enabled: false
+  with_items: "{{ __conflicting_services }}"
+  when:
+    - firewall_disable_conflicting_services
+    - item + ".service" | string in ansible_facts.services
+    - ansible_facts.services[item + ".service"]["status"] == "enabled"
```

Comment:
note that the use of `.service` implies a systemd system - I think this is ok for now

Reply:
I will change this condition to account for other init systems where the service is simply named {{ item }}

Reply:
Will push changes resolving all of the above issues, my local integration tests are currently stalling on firewalld installation

Reply:
> Will push changes resolving all of the above issues, my local integration tests are currently stalling on firewalld installation

What platform?

Reply:
Reverting my changes to when tests did not stall also did not work. Maybe its an external issue?

Reply:
Verbatim command I used for testing was `tox -e qemu-ansible-test-2.14 -- --image-name centos-8 tests/tests_default.yml`
Same issue happened when I ran it with the centos-9 image.

I think the mirror the role was trying to use was just being particularly difficult.